tools/xcodeproj_shims/installers/lldb-settings.sh

```diff
@@ -46,6 +46,23 @@ for fsp in $FRAMEWORK_SEARCH_PATHS; do
     LLDB_SWIFT_EXTRA_CLANG_FLAGS+=(" -F$fsp")
   fi
 done
+# So LLDB is aware of all headers that are not visible
+# from $FRAMEWORK_SEARCH_PATHS above
+#
+# Fox example, test hosts that depend on objc -> swift
+# bridging headers that only exists in .hmap files
+for hdr in $HEADER_SEARCH_PATHS; do
```

Comment:
To me it feel more like we just need to pump all build settings to lldb side... which makes sense because we try to mimic how xcode pass build settings within XCode? 
I feel that we need a readme (at the top of this file?) to indicate what is carried over under what conditions (such as don't look for things under DerivedData), and maybe a list of are the know build settings that might needed to be copied over?

Reply:
The problem with putting too much info here is that this runs on every build. Had performance problems in the past here specially when building really big apps.

Agree that the end goal is what you described, but I think we should do it iteratively as we see the need for new settings and keep an eye on performance as we go

I plan do add better tests for this soon to catch regressions, will make sure there's some documentation also being added 👍